homeassistant/components/ezviz/__init__.py

```diff
@@ -1 +1,120 @@
-"""Support for Ezviz devices via Ezviz Cloud API."""
+"""Support for Ezviz camera."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from pyezviz.client import EzvizClient
+from requests import ConnectTimeout, HTTPError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_REGION, CONF_TIMEOUT, CONF_USERNAME
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import (
+    ATTR_SERIAL,
+    CONF_FFMPEG_ARGUMENTS,
+    DATA_COORDINATOR,
+    DATA_UNDO_UPDATE_LISTENER,
+    DEFAULT_FFMPEG_ARGUMENTS,
+    DEFAULT_TIMEOUT,
+    DOMAIN,
+)
+from .coordinator import EzvizDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)
+
+PLATFORMS = ["camera", "sensor", "binary_sensor", "switch"]
```

Comment:
Please sort 🔡.

Reply:
VSCode's linter only allow met to sort this way. (auto corrects. to this format or 1 line)

PLATFORMS = [
    "camera",
    "sensor",
    "binary_sensor",
    "switch",
]

Hope this is acceptable?

Reply:
I mean that `"binary_sensor"` should go before `"camera"`.

Reply:
Done.